schema/project-level/project-schema.json

```
@@ -296,6 +296,31 @@
       ],
       "default": "en",
       "minLength": 1
+    },
+    "environment": {
+      "title": "Environment",
+      "description": "Information about the environmental and climate impact of the project and actions the project will take to mitigate potentially negative impacts on adjacent populations and habitats.",
+      "type": "object",
+      "properties": {
+        "goals": {
+          "title": "Environmental goals",
+          "description": "The environmental goals pursued, using the closed environmentalGoal codelist.",
```

Add link to codelist reference.

```suggestion
          "description": "The environmental goals pursued, using the closed [environmentalGoal codelist](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#environmentalgoal).",
```